xla/service/gpu/gemm_rewriter.cc

```diff
@@ -1846,7 +1846,7 @@ class GemmRewriterVisitor : public DfsHloRewriteVisitor {
                         MatrixIsColumnMajor(instr, gemm_backend_config));
 
     if (std::holds_alternative<se::RocmComputeCapability>(gpu_version_)) {
-      if (!output_is_column_major) return false;
+      //if (!output_is_column_major) return false;
```

Comment:
Why commenting this out?

Reply:
This is basically the same change as was done by Wen in https://github.com/openxla/xla/pull/5852 in order to enable row-major  hipblas-lt support (to test my impl). But now as soon as PR 5852 is merged, I can rebase and remove this "fix".